documentation/website/package.json

```diff
@@ -71,5 +71,10 @@
     "minimatch": "^3.0.4",
     "node-sass": "^6.0.1",
     "yaml": "^1.10.2"
+  },
```

Comment:
`dependencies` -> `devDependencies`

Reply:
Argh, nuts. I only fixed it for files that had conflicts. Will fix